recipes/superlance/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "superlance" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "f697c71341e9a686f3a0ff3f04a82448523eac0a6121484933729ba65a973a63" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - supervisor
+
+test:
+  imports:
+    - superlance
+    - superlance.tests
+
+about:
+  home: http://github.com/supervisor/superlance
+  license: BSD-variant
```

Comment:
What is the text they are using to define? If they say BSD-variant or ZPL-variant, let us use that. If not just use 'superlance custom'.